vendor/src/github.com/docker/engine-api/types/registry/registry.go

```diff
@@ -82,7 +82,7 @@ type SearchResult struct {
 	IsOfficial bool `json:"is_official"`
 	// Name is the name of the repository
 	Name string `json:"name"`
-	// IsOfficial indicates whether the result is trusted
+	// IsTrusted indicates whether the result is trusted
```

Comment:
@aaronlehmann This was `IsTrusted`, which changed to `IsTrusted` in the code, but had to stay `is_trusted` on the wire. I am not sure about the deprecation status, but let's do so.